examples/p4est_2d_dgsem/elixir_euler_supersonic_cylinder.jl

```diff
@@ -133,6 +133,7 @@ stage_limiter! = PositivityPreservingLimiterZhangShu(thresholds=(5.0e-7, 1.0e-6)
 
 ###############################################################################
 # run the simulation
-sol = solve(ode, SSPRK43(stage_limiter!),
-            save_everystep=false, callback=callbacks);
+sol = solve(ode, SSPRK43(stage_limiter!), abstol=1.0e-4, reltol=1.0e-4,
+            save_everystep=false, callback=callbacks,
+            internalnorm=ode_norm);
```

Comment:
```suggestion
sol = solve(ode, SSPRK43(stage_limiter!), internalnorm=ode_norm,
            save_everystep=false, callback=callbacks);
```

We shouldn't change the tolerances here (since this setup can be quite sensitive and we would like to keep the previous working version).

Reply:
Yes, sorry. That was an artifact of my tests.